src/infrastructure/MosaicHttp.ts

```diff
@@ -84,9 +83,9 @@ export class MosaicHttp extends Http implements MosaicRepository {
                             PublicAccount.createFromPublicKey(mosaicInfoDTO.mosaic.ownerPublicKey, networkType),
                             mosaicInfoDTO.mosaic.revision,
                             new MosaicProperties(
```

Comment:
The reason I kept it was because the individual for now. But will probably refactor it properly. there's a already a ticket I created for the refactorting